docs/source/storage.rst

```diff
@@ -19,3 +19,32 @@ Local storage backend is able to read data from a file or standard input and wri
 Input data can be in a form of JSON (one entry per line) or plain text (simplified JSON object resembling the ES entry described above is constructed). We also support common logging format ["timestamp  severity    message"]
 
 
+LocalDir
+--------
+Works same as local storage except will let you read from a directory of logs can be json or common log. We support only file ending with '.log' or '.json'
```

Comment:
A few typos here, I would change this to: 
*This* works the same as *the* local storage except *this* will let you read from a directory of logs *which* can *either* be json or common log. We support only *files* ending with '.log' or '.json'